VMEncryption/main/Common.py

```diff
@@ -47,7 +47,9 @@ class CommonVariables:
     encryption_settings_counter_file = 'counter.txt'
     encryption_settings_counter_path = encryption_key_mount_point + '/' + encryption_settings_counter_file
 
-    wireserver_endpoint = "http://169.254.169.254:80/machine?comp=diskEncryptionData"
+    static_wireserver_IP = '168.63.129.16'
```

Comment:
consider adding a code comment pointing to this documentation on 168.63.129.16 
https://docs.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16